File: StaticSharp/Modifiers/BackdropBlur/BackdropBlur.cs
using StaticSharp.Gears;
using System.Runtime.CompilerServices;

namespace StaticSharp {

    public interface JBackdropBlur : JAbstractBackdropFilter {
        double Radius { get; set; }
    }

    [ConstructorJs]
    public partial class BackdropBlur : AbstractBackdropFilter {
        
    }


}

File: StaticSharp/Modifiers/BackdropBlur/BackdropBlur.js
StaticSharpClass("StaticSharp.BackdropBlur", (modifier, element) => {
    StaticSharp.AbstractBackdropFilter(modifier, element)

    modifier.Reactive = {
        Radius: 5
    }

    modifier.getFilter = function () {
        return `blur(${modifier.Radius}px)`
    }  
})